core/bootstrap/bootstrap.go

```diff
@@ -60,9 +60,9 @@ var DefaultBootstrapConfig = BootstrapConfig{
 	ConnectionTimeout: (30 * time.Second) / 3, // Perod / 3
 }
 
-func BootstrapConfigWithPeers(pis []pstore.PeerInfo) BootstrapConfig {
+func BootstrapConfigWithPeers(pis []peer.AddrInfo) BootstrapConfig {
```

Comment:
func name will be used as bootstrap.BootstrapConfigWithPeers by other packages, and that stutters; consider calling this ConfigWithPeers